backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/apiaudit/apiAuditPUT.feature

```diff
@@ -76,66 +72,61 @@ Feature: API Audit PUT http method
 	"messageBody": "\"test-endpoint was called\""
   }
 """
-    And there are added records in API_Request_Audit
+    And after not more than 30s, there are added records in API_Request_Audit
       | Method | Path                                                                         | AD_User.Name | Status |
       | PUT    | api/v2/test?responseBody=%22test-endpoint%20was%20called%22&responseCode=404 | metasfresh   | Fehler |
 
-    And there are added records in API_Request_Audit_Log
+    And after not more than 30s, there are added records in API_Request_Audit_Log
       | Logmessage                                | AD_Issue.Summary               |
       | Endpoint invoked; returning httpCode: 404 | null                           |
       | Endpoint invoked; log ad_issue            | Endpoint invoked; log ad_issue |
 
-    And there are added records in API_Response_Audit
+    And after not more than 30s, there are added records in API_Response_Audit
       | HttpCode | Body                                           |
       | 404      | {"messageBody":"\"test-endpoint was called\""} |
 
   @from:cucumber
-  @ignore("notworking")
   Scenario: Testcase 130, failing PUT and caller does not wait for result
     And the following API_Audit_Config record is set
-      | Identifier | SeqNo | OPT.Method | OPT.PathPrefix | IsInvokerWaitsForResult |
-      | c_1        | 10    | PUT        | api/v2/test    | N                       |
+      | Identifier | SeqNo | OPT.Method | OPT.PathPrefix | IsInvokerWaitsForResult | IsSynchronousAuditLoggingEnabled | IsWrapApiResponse |
+      | c_1        | 10    | PUT        | api/v2/test    | N                       | Y                                | Y                 |
 
     When invoke 'PUT' '/api/v2/test?delaymillis=1000&responseBody=%22test-endpoint%20was%20called%22&responseCode=404' with response code '202'
 
-    And there are added records in API_Request_Audit
+    And after not more than 0s, there are added records in API_Request_Audit
```

Comment:
same as above. in fact, please change all "after not more than 0s" step defs. no need to have elaborate comments everywhere though (but ofc, feel free :-) )